### PR TITLE
glaze 1.0.1 (new cask)

### PR DESCRIPTION
**Pull request by pthariensflame**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

NOTE: A cask *named* `glaze` was already refused in 2016 (#28197), but this is a completely unrelated app.

### CODE REVIEW

File: Casks/glaze.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  livecheck do
+    url "https://glaze.cs.uchicago.edu/downloads.html"
+    strategy :page_match
+    regex(/Glaze\s+(\d+(?:\.\d+)*),\s+MacOS/i)
+  end
+
+  app "Glaze.app"
+
+  zap trash: '~/.glaze'
```
Comment (pthariensflame) [inline Casks/glaze.rb:26]: I don't understand the error here, given that local `style` and `audit` didn't flag it, and the [Cask Cookbook](https://docs.brew.sh/Cask-Cookbook#stanza-uninstall) contradicts it:
> [trash:](https://docs.brew.sh/Cask-Cookbook#uninstall-trash) (string or array) - single-quoted, absolute paths of files or directory trees to move to Trash

Comment (bevanjkay) [reply Casks/glaze.rb:26]: @pthariensflame Double-quoted strings are correct. The Cask Cookbook would not have been updated when our style changed from single to double-quoted strings a couple of years ago.

Comment (pthariensflame) [reply Casks/glaze.rb:26]: Thanks for the clarification!  I'll change it.

**Review by miccal — APPROVED** [review]

**Comment by miccal** [comment]
Thank you @pthariensflame.

**Pull request merged**